logictest: increase closed ts in buffered_writes further

yuzefovich opened the pull request:
We still see a RETRY_SERIALIZABLE error on COMMIT of a txn that reads large blobs under race, so increase the closed ts interval even further (beyond the test timeout).

Fixes: #145891.
Fixes: #145963.
Fixes: #146092.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/146136)
<!-- Reviewable:end -->

stevendanna reviewed (approved):
This seems reasonable to me.

It still seems like the most likely cause is the closed timestamp since nothing else should be reading these keys and bumping the timestamp cache as far as I can tell.  Maybe we should open an issue to come back to this at some point.

yuzefovich committed:
logictest: increase closed ts in buffered_writes further

We still see a RETRY_SERIALIZABLE error on COMMIT of a txn that reads
large blobs under race, so increase the closed ts interval even further
(beyond the test timeout).

Release note: None
